fathom/src/core.rs

```diff
@@ -185,6 +185,11 @@ pub enum Term<'arena> {
     ),
 }
 
+#[test]
+fn term_size() {
+    assert_eq!(std::mem::size_of::<Term>(), 56);
+}
+
```

Comment:
I think I tend to prefer to put unit tests at the end of a file, in a submodule - eg. https://github.com/yeslogic/fathom/blob/9cbfe12a1e102b06118f1299beae908ce0fc4872/fathom/src/core.rs#L707-L716